### ISSUE TITLE
"Copyright text" in about Xamarin Workbooks is not having 4.5:1 color contrast ratio.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#278]
### Steps to Reproduce

1. Launch Xamarin Work application. It will open "Create a new workbook" window.
2. Navigate to Xamarin Workbooks-> About Xamairn workbooks and activate it.
3. Check the colour contrast ration of "Copy right" text.

### Expected Behavior
Colour contrast ratio should be minimum 4.5:1

### Actual Behavior
Colour contrast ratio is less than 4.5:1
as shown in the attached screen-shots.

### User Impact:
Low vision user will feel difficulty in seeing the text.

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9) 

### Log File

![about xamarin](https://user-images.githubusercontent.com/22956130/39864253-91a588f6-5438-11e8-9067-df6c45846e37.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1835
